### ISSUE TITLE
Auth page error

### ISSUE THREAD

**Issue by yacinbenhamida** [adminkit/adminkit#6]
An error in the log of the page "pages-sign-in.html" also signup page of your template is visible in the console log with the following stack trace:
link :  https://demo.adminkit.io/pages-sign-up.html
```
Uncaught TypeError: Invalid value used as weak map key
    at WeakMap.set (<anonymous>)
    at t.e.init (simplebar.esm.js:401)
    at new t (simplebar.esm.js:354)
    at HTMLDocument.<anonymous> (sidebar.js:6)
```
this causes crashes in javascript sometimes.
browser is the brand new chromium edge, tested also on google chrome same error  "Invalid value used as weak map key"

**Comment by paullaros** [comment]
Thanks! We'll address this issue in the next release.

**Comment by paullaros** [comment]
Issue has been resolved in version 2.0.1 #12
